### DISCUSSION TITLE
The metamask signature cannot be activated normally on the mobile terminal

### DISCUSSION THREAD

**Discussion by liulise** [rainbow-me/rainbowkit#1391] [Q&A] [answered]
Hello, the official example "with-next-siwe-next-auth" is ok for me on chrome, but when I use my mobile phone to access, I can't invoke the metamask signature on my mobile phone when I click connect. I don't know why, please request help

**Comment by DanielSinclair** [comment, marked as answer]
Hi. This potentially has to do with WalletConnect v2. Ensure that you've supplied a valid `projectId` to RainbowKit and your wallet list, and you've updated to the most recent releases of Wagmi and RainbowKit. There are known issues in specific regions or behind VPNs where WalletConnect v2 will quietly fail, so if you're still experiencing the issue after trying all of the above, please open an issue and we can further investigate.
